src/EA.Weee.RequestHandlers/Mappings/AdminEvidenceNoteDataMap.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace EA.Weee.RequestHandlers.Mappings
+{
+    using System.Linq;
+    using EA.Prsd.Core;
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Core.Admin;
+    using EA.Weee.Core.Helpers;
+    using EA.Weee.Core.Scheme;
+    using EA.Weee.Domain.AatfReturn;
+    using EA.Weee.Domain.Evidence;
+    using Core = Core.AatfEvidence;
+    using Scheme = Domain.Scheme.Scheme;
+
+    public class AdminEvidenceNoteDataMap : IMap<Note, AdminEvidenceNoteData>
```

Comment:
sure, it could be re-used. I have asked yesterday shall I move the EvidenceNoteData in the admin/shared area so it's used across but you said it has to stay in the Aatf evidence area.

Reply:
Sorry a bit of a misunderstanding. It can stay in the AATF evidence folder but can be re-used across any request handlers. The folders just keep similar core classes together.

Reply:
okay. i will make the changes

Reply:
got rid of AdminEvidenceNoteData class and its related mappers